contrib/cirrus/setup.sh

```diff
@@ -14,7 +14,17 @@ if [[ -r "/etc/ci_environment" ]]; then
 fi
 trap "complete_setup" EXIT
 
+
 msg "************************************************************"
 msg "Setting up runtime environment"
 msg "************************************************************"
+
+dnf -y install make automake gcc gcc-c++ kernel-devel
```

Comment:
Ref: Added into package list after this PR merged, will get picked up in next image update: https://github.com/containers/automation_images/pull/117